docs/config.md

```diff
@@ -53,6 +53,22 @@ Overrides `CARTESI_AUTH_AWS_KMS_*`.
 
 * **Type:** `string`
 
+## `CARTESI_AUTH_PRIVATE_KEY`
+
+The node will use this private key to sign transactions.
+
+Overrides `CARTESI_AUTH_PRIVATE_KEY_FILE`, `CARTESI_AUTH_MNEMONIC`, `CARTESI_AUTH_MNEMONIC_FILE` and `CARTESI_AUTH_AWS_KMS_*`.
+
+* **Type:** `string`
+
+## `CARTESI_AUTH_PRIVATE_KEY_FILE`
+
+The node will use the private key contained in this file to sign transactions.
+
+Overrides `CARTESI_AUTH_MNEMONIC`, `CARTESI_AUTH_MNEMONIC_FILE` and `CARTESI_AUTH_AWS_KMS_*`.
```

Comment:
`Overrides `CARTESI_AUTH_MNEMONIC_*` and `CARTESI_AUTH_AWS_KMS_*`.`

Reply:
I think it's more clear the way I did instead of using `*`

Reply:
I was going to suggest to than remove the `*` from the `KMS_*`, but @gligneul told me this description is going to change in his `config` PR, so best to leave it as it is.